src/web/app/pages-admin/admin-home-page/admin-home-page.component.ts

```diff
@@ -29,6 +21,7 @@ export class AdminHomePageComponent {
   instructorInstitution: string = '';
 
   instructorsConsolidated: InstructorData[] = [];
+  isInstructorRowEditModeEnabled: boolean[] = [];
```

Comment:
Possible to just include this as a new field in `InstructorData`?

Reply:
Yes, you're right. Changed.